pkg/tsdbctl/query.go

```diff
@@ -100,13 +99,8 @@ Arguments:
 		"Query filter, as an Iguazio Continuous Data Platform\nfilter expression. To reference a metric name from within\nthe query filter, use the \"__name__\" attribute.\nExamples: \"method=='get'\"; \"__name__='cpu' AND os=='win'\".")
 	cmd.Flags().StringVarP(&commandeer.last, "last", "l", "",
 		"Return data for the specified time period before the\ncurrent time, of the format \"[0-9]+[mhd]\" (where\n'm' = minutes, 'h' = hours, and 'd' = days>). When setting\nthis flag, don't set the -b|--begin or -e|--end flags.\nExamples: \"1h\"; \"15m\"; \"30d\" to return data for the last\n1 hour, 15 minutes, or 30 days.")
-	cmd.Flags().StringVarP(&commandeer.windows, "windows", "w", "",
-		"Overlapping windows of time to which to apply the aggregation\nfunctions (if defined - see the -a|--aggregates flag), as a\ncomma separated list of integer values (\"[0-9]+\").\nThe duration of each window is calculated by multiplying the\nvalue from the windows flag with the aggregation interval\n(see -i|--aggregation-interval). The windows' end time is\nthe query's end time (see -e|--end and -l|--last). If the\nwindow's duration extends beyond the query's start time (see\n-b|--begin and -l|--last), it will be shortened to fit the\nstart time. Example: -w \"1,2\" with -i \"2h\", -b 0, and the\ndefault end time (\"now\") defines overlapping aggregation\nwindows for the last 2 hours and 4 hours.")
-	// The default aggregates list for an overlapping-windows query is "avg",
-	// provided the TSDB instance has the "count" and "sum" aggregates, which
-	// make up the "avg" aggregate; ("count" is added automatically when adding
-	// any other aggregate). However, it was decided that documenting this
-	// would over complicate the documentation.
+	cmd.Flags().StringVarP(&commandeer.aggregationWindow, "aggregation-window", "w", "",
+		"Sliding window to calculate the aggregation for every step. Examples: \"1h\"; \"150m\". (default = 0)")
```

Comment:
The part about every step is not really clear. What does it mean for the window to be 0 (the default)?